pkg/datapath/loader/tc.go

```diff
@@ -48,20 +56,28 @@ func attachSKBProgram(device netlink.Link, prog *ebpf.Program, progName, bpffsDi
 	}
 
 	// Legacy tc attached, make sure tcx is detached in case of downgrade.
+	// netkit can only be used in combination with tcx, but never legacy tc,
```

Comment:
As the comment above suggests, I think we should never get to this code if we attach to a netkit device.

Reply:
Yes, that is correct.